utils/update_checkout/update-checkout-config.json

```diff
@@ -41,7 +41,7 @@
             "platforms": [ "Linux" ]
         },
         "yams": {
-            "remote": { "id": "jpsim/Yams" }
+            "remote": { "id": "asuhan/Yams" }
```

Comment:
@ematejska Required until https://github.com/jpsim/Yams/pull/260 (or a better version of it) is merged and released.

Reply:
Got it.